.github/workflows/test-python.yml

```diff
@@ -46,4 +46,4 @@ jobs:
           pip install -e ".[zstd, srv]"
       - name: Run mypy
         run: |
-          mypy --install-types --non-interactive bson gridfs tools
+          mypy --install-types --non-interactive bson gridfs tools test
```

Comment:
Does it make sense to separate the test/ type checking so that we can globally ignore certain error codes in the test/ package? Eg:
```
mypy --install-types --non-interactive bson gridfs tools
mypy --install-types --non-interactive --disable-error-code var-annotated --disable-error-code attr-defined --disable-error-code union-attr --disable-error-code assignment test
```

Disabling some checks could make the tests easier to maintain.

Reply:
Done